src/main/java/io/kestros/commons/structuredslingmodels/services/impl/BaseValidationProviderService.java

```diff
@@ -194,4 +198,9 @@
         break;
     }
   }
+  
+  private <T> T castModel(Class<T> clazz, Object model)
```

Comment:
The model parameter is not used and shows up as an inspection warning in Intelliij.

Reply:
updated here https://github.com/kestros/kestros-structured-sling-models/pull/35/files